salt/orch/update.sls

```diff
@@ -116,13 +117,13 @@ admin-setup:
 # with the real update.
 pre-orchestration-migration:
   salt.state:
-    - tgt: '{{ is_regular_node_tgt }}'
+    - tgt: '{{ is_updateable_node_tgt }}'
```

Comment:
Note that there is a change in the target for these migrations, from _"all the master and minion machines in the cluster"_ to _"all the **updateable** master and minion machines in the cluster"_. Not sure about the consequences of this change.

Reply:
Yes, this was a side effect discovered by @kiall. It's also a rather artificial case, but it boils down to: only rename nodes that are going to be part of the update process. Otherwise you'll end up with a `kubectl get nodes` listing that could have `NotReady` new entries that weren't going to be migrated in any moment.

The way to reproduce as I say is pretty artificial (e.g. you run transactional-update only on the masters, and apply the upgrade, all machines will get renamed without this change).

Reply:
Ok, I see.